apps/aevm/test/aevm_test_utils.erl

```diff
@@ -300,6 +300,7 @@ build_data_array([], Acc) ->
                                 , {post, leave_out}
                                 , {sourceHash, default, <<>>}
                                 , {lllcversion, default, <<>>}
+                                , {off_chain, default, false}
```

Comment:
I think you rather wanted to amend DefaultEnv in get_config/1 ? This seems the format of the external test JSON data.